src/controllers/resize-controller.js

```diff
@@ -33,19 +33,25 @@ export class ResizeController {
     host.addController(this);
 
     /** @type {number|null} the latest observed `width` of the host */
-    this.width = null;
+    this._width = null;
 
     this._host = host;
 
-    /** @type {Function|undefined} if defined, the callback is executed everytime a resize happens */
     this._callback = options?.callback;
 
-    /** @type {number[]|undefined} if defined sets attributes `w-lt-${breakpoint}` & `w-gte-${breakpoint}` for each breakpoint */
     this._widthBreakpoints = options?.widthBreakpoints;
 
     this._observer = new ResizeObserver(() => this._onResize());
   }
 
+  get width () {
+    return this._width;
+  }
+
+  set width (newWidth) {
+    this._width = newWidth;
+  }
+
   _onResize () {
     // NOTE: We could use entries[0].borderBoxSize.inlineSize but not supported in Chrome, Safari or polyfill
     this.width = this._host.getBoundingClientRect().width;
```

Comment:
fix: use the private property directly
```suggestion
    this._width = this._host.getBoundingClientRect().width;
```